fury/utils.py

```diff
@@ -880,3 +880,27 @@ def normals_from_v_f(vertices, faces):
     norm[faces[:, 2]] += n
     normalize_v3(norm)
     return norm
+
+def vertices_from_actor(actor):
+
+    """ This function will return vertices from actor
+    """
+    vtkPoint = actor.GetMapper().GetInput()
+    points = vtkPoint.GetPoints()
```

Comment:
Can you do it in one line: ` points =  actor.GetMapper().GetInput().GetPoints()`